hack/.linted_packages

```diff
@@ -388,7 +388,6 @@ staging/src/k8s.io/kube-aggregator/pkg/client/listers/apiregistration/v1alpha1
 staging/src/k8s.io/sample-apiserver
```

Comment:
Ah, OK. I assumed this some sort of automagically-generated updated that didn't get propagated.